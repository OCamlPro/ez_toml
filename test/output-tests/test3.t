  $ dune exec -- toml-check --error E-21 test3.toml
  Parsing "test3.toml"
  {
    "comments": {
      "a": 1,
      "x": 1,
      "y": 1
    },
    "extensions": {
      "w": 1,
      "x": 2,
      "z": {
        "b": 1
      }
    }
  }
  $ dune exec -- toml-check --error E-21 --to-toml test3.toml
  Parsing "test3.toml"
  
  # Extensions on operators
  
  [extensions]
  
  w = 1
  x = 2
  [extensions.z]
  b = 1
  
  # Comments
  
  [comments]
  
  # This is a comment on field x
  
  x = 1
  
  # these two comments are separated
  
  # by an empty line
  
  y = 1
  
  # and finally, this field has both a comment before
  
  a = 1 # and after the definition
  
