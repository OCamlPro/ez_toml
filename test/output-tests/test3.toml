
# Extensions on operators

[extensions]

w = 1
x = 1
y = 1
z = { a = 1, b = 1 }

w == 2
x := 2
{} -= [ "y" ]
z -= [ "a" ]

# Comments

[comments]

# This is a comment on field x

x = 1

# these two comments are separated

# by an empty line

y = 1

# and finally, this field has both a comment before

a = 1 # and after the definition
