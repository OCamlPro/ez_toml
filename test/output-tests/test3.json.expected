{
  "comments": {
    "a": 1,
    "x": 1,
    "y": 1
  },
  "extensions": {
    "w": 1,
    "x": 2,
    "z": {
      "b": 1
    }
  }
}
